src/RA_Dlg_Memory.cpp

```diff
@@ -881,16 +880,19 @@ INT_PTR Dlg_Memory::MemoryProc(HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPara
                     {
                         if (pDIS->itemID == 0)
                         {
-                            const std::string& sFirstLine = m_SearchResults[m_nPage].m_results.Summary();
-                            if (sFirstLine.empty())
-                                _stprintf_s(buffer, sizeof(buffer), _T("%s"), _T("Invalid Range"));
-                            else
+                            try
+                            {
+                                const std::string& sFirstLine = m_SearchResults.at(m_nPage).m_results.Summary();
                                 _stprintf_s(buffer, sizeof(buffer), _T("%s"), NativeStr(sFirstLine).c_str());
+                            } catch (const std::out_of_range& e)
+                            {
+                                _stprintf_s(buffer, sizeof(buffer), _T("%s"), _T(e.what()));
```

Comment:
can you really convert `e.what()` like that? I thought _T only applied to literals.